it/services/GetFinancialDetailsServiceISpec.scala

```diff
@@ -30,113 +30,38 @@ class GetFinancialDetailsServiceISpec extends IntegrationSpecCommonBase with ETM
-          lastClearedAmount = Some(123.45),
-          statisticalFlag = true,
-          informationCode = Some("A"),
-          paymentLot = Some("81203010024"),
-          paymentLotItem = Some("000001"),
-          interestRate = Some(543.21),
-          interestFromDate = Some(LocalDate.of(2022, 10, 30)),
-          interestEndDate = Some(LocalDate.of(2022, 10, 30)),
-          latePaymentInterestID = Some("1234567890123456"),
-          latePaymentInterestAmount = Some(123.45),
-          lpiWithDunningBlock = Some(543.21),
-          accruingPenaltyLPP1 = Some("Interest Rate")
-        ))
-      ),
-      financialDetails = Seq(FinancialDetails(
-        documentId = "0001",
-        taxPeriodFrom = Some(LocalDate.of(2022, 10 ,30)),
-        taxPeriodTo = Some(LocalDate.of(2022,10,30)),
-        items = Seq(FinancialItem(
-          dueDate = Some(LocalDate.of(2022, 10 ,30)),
-          clearingDate = Some(LocalDate.of(2022, 10 ,30)),
-          metadata = FinancialItemMetadata(
-            subItem = Some("001"),
-            amount = Some(123.45),
-            clearingReason = Some("01"),
-            outgoingPaymentMethod = Some("outgoing Payment"),
-            paymentLock = Some("paymentLock"),
-            clearingLock = Some("clearingLock"),
-            interestLock = Some("interestLock"),
-            dunningLock = Some("dunningLock"),
-            returnFlag = Some(true),
-            paymentReference = Some("Ab12453535"),
-            paymentAmount = Some(543.21),
-            paymentMethod = Some("Payment"),
-            paymentLot = Some("81203010024"),
-            paymentLotItem = Some("000001"),
-            clearingSAPDocument = Some("3350000253"),
-            codingInitiationDate = Some(LocalDate.of(2022, 10 ,30)),
-            statisticalDocument = Some("S"),
-            DDCollectionInProgress = Some(true),
-            returnReason = Some("ABCA"),
-            promisetoPay = Some("promisetoPay")
-          )
-        )),
-        originalAmount = Some(123.45),
-        outstandingAmount = Some(543.21),
-        mainTransaction = Some(MainTransactionEnum.VATReturnFirstLPP),
-        chargeReference = Some("XM002610011594"),
-        metadata = FinancialDetailsMetadata(
-          taxYear = "2022",
-          chargeType = Some("PAYE"),
-          mainType = Some("2100"),
-          periodKey = Some("13RL"),
-          periodKeyDescription = Some("abcde"),
-          businessPartner = Some("6622334455"),
-          contractAccountCategory = Some("02"),
-          contractAccount = Some("X"),
-          contractObjectType = Some("ABCD"),
-          contractObject = Some("00000003000000002757"),
-          sapDocumentNumber = Some("1040000872"),
-          sapDocumentNumberItem = Some("XM00"),
-          subTransaction = Some("5678"),
-          clearedAmount = Some(123.45),
-          accruedInterest = Some(543.21)
-        )
+
+    val getFinancialDetailsModel: FinancialDetails = getFinancialDetails.FinancialDetails(
```

Comment:
Do you need the getFinancialDetails prefixes?

Reply:
Removed ✅